configs/12.0/distros/ubuntu-14.mk

```diff
@@ -70,7 +70,8 @@ ifndef AT_DISTRO_REQ_PKGS
     AT_COMMON_PKGS_REQ := zlib1g-dev libncurses5-dev ncurses-term flex bison \
                           texinfo subversion cvs gawk fakeroot debhelper \
                           autoconf rsync curl bc libxml2-utils automake \
-                          docbook2x dpkg-sig xutils-dev wget autoconf-archive
+                          docbook2x dpkg-sig xutils-dev wget autoconf-archive \
+                          make-[4-9]
```

Comment:
@er-1  If make-[4-9]  means make-4 to make-9 .. why should we restrict till make-9 ? I think our intention is to check make 4+ .

Reply:
I do agree this is the correct solution, but I don't think we need to be over-engineer this.
I suggest the following:
1. Remove ubuntu-14.mk.  We don't support it on AT >= 12.0.  We don't even test it.
2. For SLES 12 and Ubuntu 16.04, we can have just `make` because these distros already provide a GNU make that's newer than 4.0.
3. If you really believe that we need to specify a version there, `[4-9]` should be a large enough window, i.e. Ubuntu 16.04 will likely reach EOL before GNU make 10.0 is released.  ;-)

Reply:
@tuliom Ack

Reply:
@tuliom I'll remove ubuntu-14.mk. What about centos-6.mk and centos-7.mk? I guess I could remove them too.